frontend/tailwind.config.js

```diff
@@ -1,32 +1,58 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    './pages/**/*.{js,ts,jsx,tsx,mdx}',
-    './components/**/*.{js,ts,jsx,tsx,mdx}',
-    './app/**/*.{js,ts,jsx,tsx,mdx}',
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
looks ok but it might be worth to investigate why do you have this many changes in the files - here I can see a discrepancy between quotes style. It would be worth to investigate and align with the team on the approach and then have a local config file in the project to make sure everybody uses the same linting rules